jenkins/branch.name

```diff
@@ -1 +1 @@
-main
+no-current-env
```

Comment:
I imagine that you don't want this change merged...is that correct?

Reply:
Correct, I didn't want to pollute the current image on quay.io.  I have added a "DO NOT MERGE" label.

Reply:
And I have pulled the change out into a separate commit to make it easy to isolate and remove.